src/lualib/ArrayIncludes.ts

```diff
@@ -0,0 +1,22 @@
+// https://www.ecma-international.org/ecma-262/9.0/index.html#sec-array.prototype.includes
+function __TS__ArrayIncludes<T>(this: T[], searchElement: T, fromIndex = 0): boolean {
+    const len = this.length;
+    let k = fromIndex;
+
+    if (fromIndex < 0) {
+        k = len + fromIndex;
+    }
+
+    if (k < 0) {
+        k = 0;
+    }
+
+    while (k < len) {
```

Comment:
Thoughts on using `forRange` there?

Reply:
I like the `while` because it is closer to both the ecmascript standard, and to the lua that will be generated. But I think forRange is faster, so I changed it.